client/src/app/styles/App.css

```diff
@@ -407,7 +407,7 @@ template {
  * Set up a decent box model on the root element
  */
 .primary-button {
-  background-color: #2A363B;
+  background-color: #5cc98b;
   width: 100%;
```

Comment:
in the future, make colours as Sass variables, that way it will be easy to change theme colours if we ever need to